packages/block-library/src/embed/core-embeds.js

```diff
@@ -227,13 +245,14 @@ export const others = [
 		patterns: [ /^http:\/\/g?i*\.photobucket\.com\/.+/i ],
 	},
 	{
+		// Deprecated in favour of the core-embed/crowdsignal block
```

Comment:
I'm not sure if we can remove the pattern because of backward compatibility. However we can hide the deprecated version from the inserter. We did that for subheading block and the old version of columns block.

Reply:
Removing the patterns here is fine, they're only used to resolve URLs into the correct block, so existing blocks are loaded, and converted to the Crowdsignal block without a problem.

I'm not sure we should hide the old block either. It's useful for people who still have the polldaddy name in their minds, and if they use the polldaddy block, it'll convert to a crowdsignal block automatically.

Reply:
You can resolve it by adding a keyword to the new block.

Reply:
That doesn't help people scrolling through the list instead of typing to search though.

Reply:
Yes, true. That’s the broader issue with rebranding 😅 I will leave it up to you. We should have some consistent strategy for depreciations.

Reply:
I think it's fine to remove it from the inserter, check the deprecation of `core-embed/speaker` in favour of `core-embed/speaker-deck`.

For people still looking for "Polldaddy", perhaps add "formerly Polldaddy" to the description of the new Crowdsignal block, and maybe to the name, too.

Reply:
ac09b7cab68c026cbd14c82074ea59a8d5245a97 - removed from the inserter similar to `core-embed/speaker` as suggested

Reply:
b9584b6d3a6966505da8f04d6baae318b98daa48 - added `polldaddy` as keyword to make it discoverable.

Reply:
d0b2233593265693076a8dd80325e92bdaf246e4 - added `polldaddy` keyword for better discoverability of the new block